src/pde_solve.jl

```diff
@@ -59,5 +66,87 @@ function DiffEqBase.solve(
 
     Flux.train!(loss, ps, data, opt; cb = cb)
 
-    save_everystep ? iters : u0(X0)[1]
+
+    if give_limit == false
+        save_everystep ? iters : u0(X0)[1]
+    else
+        A = prob.kwargs.data.A
+        u_domain = prob.kwargs.data.u_domain
+
+        ## UPPER LIMIT
+        m1 = trajectories_upper
+        sdeProb = SDEProblem(μ , σ , X0 , prob.tspan)
+        ensembleprob = EnsembleProblem(sdeProb)
+        sim = solve(ensembleprob, sdealg, ensemblealg, dt=dt,trajectories=m1,adaptive=false)
+        function sol_high()
+            Uo = []
+            for u in sim.u
+                xsde = u.u
+                U = g(xsde[end])
+                u = u0(X0)[1]
+                for i in length(ts):-1:3
+                    t = ts[i]
+                    _σᵀ∇u = σᵀ∇u[i-1](xsde[i-1])
+                    dW = sqrt(dt)*randn(d)
+                    U = U .+ f(xsde[i-1], U, _σᵀ∇u, p, t)*dt .- _σᵀ∇u'*dW
+                end
+                Uo = vcat(Uo , U)
+            end
+            Uo
+        end
+
+        loss_() = sum(sol_high())/m1
```

Comment:
```suggestion
        loss_() = sum(sol_high())/trajectories_upper
```

no need for two names